backend/stats/views.py

```diff
@@ -7,49 +7,51 @@
 from rest_framework import status
 from django.db.models import Count, Q
 from User.models import User
+from .user_stats_table import UserTable
+from .leader_board_table import LeaderBoardTable
+from .friend_stats_table import FirendsTable
 
+from .mixins import UserTableMixin
+from rest_framework.response import Response
 
-class StatsView(APIView):
+class StatsView(APIView, UserTableMixin):	
```

Comment:
This should probably be StatsDetailView mapped to the endpoint endpoint /stats/<user_id: int>
I don't think /stats and sending all the stats data of every user on the platform should be allowed.